addon/reducers/index.js

```diff
@@ -0,0 +1,16 @@
+// please write your own index.js that exports legit reducers
+//
+// import { combineReducers } from 'redux';
+// const foobar = (state, action) => {
+//   //reducer code here
+// };
+//
+// export default combineReducers({
+//   foobar
+// })
+
+export default function(combine) {
+  return (state, action) => {
```

Comment:
@brettburley should I add some type of console log or something here to alert the consuming app that they need to write a custom reducer?